pymc3/distributions/discrete.py

```diff
@@ -1177,6 +1161,22 @@ def logp(value, p):
         )
 
 
+class ConstantRV(RandomVariable):
+    name = "constant"
+    ndim_supp = 0
+    ndims_params = [0]
+    dtype = "floatX"  # Should be treated as a discrete variable!
+    _print_name = ("Constant", "\\operatorname{Constant}")
+
+    @classmethod
+    def rng_fn(cls, rng, c, size=None):
+        ones = np.ones_like(c)
+        return rng.randint(ones, ones + 1, size=size) * c
```

Comment:
There must be a less hacking way to get a filled array given `c` and `size`, that works as other random generators would. `np.full(size, c)` doesn't seem to cut it, because it works with shape and not with size. Is there an equivalent?

Reply:
Isn't shape and size the same thing here?

Reply:
Okay, actually I think the only problem was `size=None`. I added a check for that condition...